shaded-dependencies/build.sbt

```diff
@@ -28,9 +28,11 @@ publishArtifact in (Compile, packageSrc) := false
 
 publishArtifact in (Compile, packageBin) := false
 
-val hive_version = "3.1.1"
+val hive_version = sys.props.getOrElse("hive.version", "3.1.1")
```

Comment:
Shall we make the default 3.1.2 instead?

Reply:
done

Reply:
Should the same be done for ORC version as well ?

Reply:
for ORC its already there ..